packages/rmf-auth/package.json

```diff
@@ -28,7 +28,7 @@
     "@mui/lab": "5.0.0-alpha.86",
     "api-client": "workspace:*",
     "eventemitter3": "^4.0.7",
-    "keycloak-js": "^11.0.2"
+    "keycloak-js": "^22.0.5"
```

Comment:
any reason why it is updated to 22 and not the latest?

Reply:
if I update it to the latest or anything > 22.0.5, a significant jump in number of subdependencies will need to be updated and things start to break

I will be continuing with more tests and trying again today before merging

Reply:
yup anything above 22.0.5 requires ES6 and above, it's probably not the best idea to migrate even this branch to the latest, I'll try to test with the cluster